src/main/java/com/karankumar/bookproject/ui/settings/SettingsView.java

```diff
@@ -15,33 +15,45 @@
 
 package com.karankumar.bookproject.ui.settings;
 
+import com.karankumar.bookproject.backend.service.BookService;
 import com.karankumar.bookproject.ui.MainView;
-import com.karankumar.bookproject.ui.components.toggle.SwitchToggle;
+import com.karankumar.bookproject.ui.components.toggle.PaperToggle;
+import com.karankumar.bookproject.ui.components.dialog.ResetShelvesDialog;
 import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.button.Button;
 import com.vaadin.flow.component.html.Label;
 import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
 import com.vaadin.flow.component.orderedlayout.VerticalLayout;
 import com.vaadin.flow.dom.ThemeList;
 import com.vaadin.flow.router.PageTitle;
 import com.vaadin.flow.router.Route;
 import com.vaadin.flow.theme.lumo.Lumo;
+import lombok.extern.java.Log;
+import org.springframework.beans.factory.annotation.Autowired;
+
 
 @Route(value = "settings", layout = MainView.class)
 @PageTitle("Settings | Book Project")
+@Log
 public class SettingsView extends HorizontalLayout {
+
     private static final String ENABLE_DARK_MODE = "Enable dark mode";
     private static final String DISABLE_DARK_MODE = "Disable dark mode";
-    private static SwitchToggle darkModeToggle;
-    private static Label darkModeLabel = new Label(ENABLE_DARK_MODE);
+    private static PaperToggle darkModeToggle;
     private static boolean darkModeOn = false;
+    private static Label darkModeLabel = new Label(ENABLE_DARK_MODE);
+
+    // -------------- Clear Shelves ----------------------
```

Comment:
As per our style guide, we're not using any decorators in patterns. You can keep the 'Clear shelves' comment in, but please remove the dashes